tests/test_colors.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import sys
+from pathlib import Path
+
+import pytest
+
+from . import CASES_DIR
+from ._manim import Text
+from .svg_tester import SVGStyleTester
+
+
+def test_invalid_color_fail(tmpdir):
+    loc = Path(tmpdir, "test.svg")
+    assert not loc.exists()
+    with pytest.raises(ValueError):
+        Text("color", color="invalid", filename=str(loc))
+
+
+@pytest.mark.skipif(
+    sys.platform.startswith("win32"), reason="windows draws fonts differently"
```

Comment:
I think `SVGStyleTester` only tests for styles and not actual paths, so I think this can be run on win32 also.